lib/between_meals/repo/git.rb

```diff
@@ -24,11 +24,8 @@ class Repo
     # Git provider
     class Git < BetweenMeals::Repo
       def setup
-        if File.exists?(File.expand_path(@repo_path))
-          @repo = Rugged::Repository.new(File.expand_path(@repo_path))
-        else
-          @repo = nil
-        end
+        fail unless File.exists?(File.expand_path(@repo_path + '.git'))
```

Comment:
So maybe, just maybe, there is a good reason for it? https://github.com/facebook/grocery-delivery/issues/16 and https://github.com/facebook/grocery-delivery/pull/17